Patch to modify Makefile to set the OMP_NUM_THREADS for the test jobs coming from EasyBuild
Author: J. Sassmannshausen (Imperial College London/UK)
diff --git a/elk-11.2.3.orig/tests/test-mpi.sh b/elk-11.2.3/tests/test-mpi.sh
index 208a2e3..9aa212f 100755
--- a/elk-11.2.3.orig/tests/test-mpi.sh
+++ b/elk-11.2.3/tests/test-mpi.sh
@@ -7,7 +7,7 @@ do
   echo
   echo "Running test in directory $i..."
   \rm -f *.OUT gmon.out fort.*
-  OMP_NUM_THREADS=2 OMP_STACKSIZE=128M mpirun -n 4 ../../src/elk > test.log
+  OMP_NUM_THREADS=${OMP_NUM_THREADS} OMP_STACKSIZE=128M mpirun -n 4 ../../src/elk > test.log
   NERROR=`grep -c Error test.log`
   if test $NERROR -gt 0
   then
diff --git a/elk-11.2.3.orig/tests/test.sh b/elk-11.2.3/tests/test.sh
index 53187a5..d9f7223 100755
--- a/elk-11.2.3.orig/tests/test.sh
+++ b/elk-11.2.3/tests/test.sh
@@ -7,7 +7,7 @@ do
   echo
   echo "Running test in directory $i..."
   \rm -f *.OUT gmon.out fort.*
-  OMP_NUM_THREADS=8 OMP_STACKSIZE=128M ../../src/elk > test.log
+  OMP_NUM_THREADS=${OMP_NUM_THREADS} OMP_STACKSIZE=128M ../../src/elk > test.log
   NERROR=`grep -c Error test.log`
   if test $NERROR -gt 0
   then

